.github/workflows/terraform.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Terraform'
+
+on:
+  push:
+    branches: [ '**' ]
+    paths: [ '/terraform' ]
+
+jobs:
+  terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: hashicorp/setup-terraform@v3
+    - run: terraform fmt -check -diff -recursive
```

Comment:
Since the changes will not persist unless committed is this just to see that formatting changes need to be made?

Reply:
Yes. That's why the `-check` flag is there.